test/es6/generators-apis.js

```diff
@@ -50,6 +50,39 @@ var tests = [
         }
     },
     {
+        // Regression Test GitHub Issue #3065: Generator length property is inconsistently configurable (+ assert failure in debug build) 

```

Comment:
can we add test (if not already) for other built-ins and a non-built-in test for same scenario?